app/controllers/shared/api/auth.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use Appwrite\Auth\Auth;
+use Appwrite\Utopia\Request;
+use Utopia\App;
+use Appwrite\Extend\Exception;
+use Utopia\Database\Document;
+use Utopia\Database\Validator\Authorization;
+
+App::init()
```

Comment:
Why did we move this to a separate file ?

Reply:
Yeh, I wanted to try and minimize each hook's responsibility

Reply:
Okay sounds good. @shimonewman let's keep it that way 👍